modules/Dasboard/components/Navbar/Navbar.tsx

```diff
@@ -20,7 +22,12 @@ const Navbar = () => {
   const router = useRouter();
   const [isModalOpen, setIsModalOpen] = useState(false);
 
+  const logout = () => (dispatch: TAppDispatch) => {
+    dispatch(projectApi.util.resetApiState());
+  };
```

Comment:
Is it necessary to wrap this by a method? Can't this be directly invoked inside the handleLogin method?

Reply:
It can be done.